vassal-app/src/main/java/VASSAL/build/module/map/Zoomer.java

```diff
@@ -430,6 +429,19 @@ public LevelConfigurer(final Zoomer z, String key, String name) {
       levelField.setMaximumSize(new Dimension(
         Integer.MAX_VALUE, levelField.getPreferredSize().height));
 
+      // Edit box selects all text when first focused
+      levelField.addFocusListener(new java.awt.event.FocusAdapter() {
+        @Override
+        public void focusGained(java.awt.event.FocusEvent evt) {
+          SwingUtilities.invokeLater(new Runnable() {
```

Comment:
Does this need to be in an `invokeLater`? IIRC, your listeners for Swing GUI elements are executed from the EDT already.

Reply:
The place where I got this code from specifically had some reason for doing it "later but on EDT" that sounded correct but now I've forgotten. There's something we're making sure it gets done *after* basically.

Reply:
![image](https://user-images.githubusercontent.com/3742246/177216532-ba1f86ce-8dcf-4766-8900-15bddf754e05.png)